src/backend/llm/model.py

```diff
@@ -4,7 +4,7 @@
 from test_llm import test_text
 
 load_dotenv()
-GROQ_API_KEY = os.getenv('GROQ_API_KEY')
+GROQ_API_KEY = os.getenv('GROQ_API_KEY2')
```

Comment:
This can be changed to its original state again. I just tested a different API key.